client/extensions/woocommerce/state/sites/promotions/actions.js

```diff
@@ -13,3 +15,34 @@ export function fetchPromotions( siteId, perPage = undefined ) {
 		perPage,
 	};
 }
+
```

Comment:
should we have docblocks for these actions?

Reply:
Thanks for the reminder. I started a README for these, but didn't add the create, update, or delete actions to it. I've added them now.